lib/bolt/cli.rb

```diff
@@ -804,6 +804,20 @@ def analytics
     end
 
     def bundled_content
+      # If the bundled content directory is empty, Bolt is likely installed as a gem.
+      if ENV['BOLT_GEM'] != 'false' && gem_install?
```

Comment:
The ticket said "when BOLT_GEM is anything but true", which seemed weird as we'd need to first check that it wasn't `nil`. Since a user will need to explicitly set this variable to disable the warning, I felt that requiring the value to be `false` made sense.

Reply:
I think common practice with variables like these is to simply require they be set. That is, setting `BOLT_GEM` means "I know I'm running from a gem install and that's okay".